### PR TITLE
Update version and CHANGELOG for new release: 0.1.1.

### PR DESCRIPTION
**Pull request by xydesa**

### CODE REVIEW

File: CHANGELOG.md
Comment (tdenewiler) [inline CHANGELOG.md:20]:
```suggestion
- Pin flake8 version to less than v5 until <https://github.com/tholo/pytest-flake8/issues/87> is fixed. (#58)
```

**Review by tdenewiler — APPROVED** [review]

File: CHANGELOG.md
Comment (tdenewiler) [reply CHANGELOG.md:20]: Overall LGTM just needs a minor fix for bare URL. Approving now to not hold up merging when that fix is in.

**Review by gregtkogut — APPROVED** [review]

**Review by gregtkogut — APPROVED** [review]

File: CHANGELOG.md
Comment (denewiler) [inline CHANGELOG.md:15]:
```suggestion
## v0.1.1 - 2022-10-10
```

**Review by denewiler — APPROVED** [review]

**Pull request merged**
